iree/test/e2e/xla_ops/partial/add_f16.mlir

```diff
@@ -0,0 +1,7 @@
+func @tensor() attributes { iree.module.export } {
+  %0 = iree.unfoldable_constant dense<[1.0, 2.0, 3.0, 4.0]> : tensor<4xf16>
+  %1 = iree.unfoldable_constant dense<[5.0, 6.0, 7.0, 8.0]> : tensor<4xf16>
```

Comment:
not sure how we handle this in other tests, but if the test is specifically exercising floating point arithmetic, shouldn't the numbers used at least have some digits after the decimal point? :P 

These tests could pass even if we truncate to `i16` :P

Reply:
Added some decimals :)